examples/ping_with_response_time.cr

```diff
@@ -7,6 +7,7 @@ require "../src/discordcr"
 client = Discord::Client.new(token: "Bot MjI5NDU5NjgxOTU1NjUyMzM3.Cpnz31.GQ7K9xwZtvC40y8MPY3eTqjEIXm", client_id: 229459681955652337_u64)
 
 client.on_message_create do |payload|
+  client.modify_guild 0u64, name: "Hi"
```

Comment:
Why is the guild name being modified here? (If this was added on purpose it should be moved in a extra example.)

Reply:
Great catch. Was a test to see if it'd compile. Going to fix & squash.